src/p2p/CycleCreator.ts

```diff
@@ -739,6 +746,9 @@ function makeCycleRecord(
     archiverListHash: '',
     standbyNodeListHash: '',
     random: config.debug.randomCycleData ? Math.floor(Math.random() * 1000) + 1 : 0,
+    txadd: [],
+    txremove: [],
+    txlisthash: '',
```

Comment:
i think we should be using camel case for cycle record fields. thats the pattern for all other fields in the cycle record